pepper-apis/src/main/resources/db-changes/schema/DDP-3709-file-uploads-tables.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="ypavlov" id="20201208-file-uploads-tables">
+
+        <createTable tableName="file_upload_status">
+            <column name="file_upload_status_id" type="bigint" autoIncrement="true">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="file_upload_status_pk"/>
+            </column>
+            <column name="file_upload_status_name" type="varchar(10)">
+                <constraints nullable="false" unique="true" uniqueConstraintName="file_upload_status_name_uk"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="file_upload">
+            <column name="file_upload_id" type="bigint" autoIncrement="true" startWith="1">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="file_upload_pk"/>
+            </column>
+            <column name="file_upload_guid" type="varchar(10)">
+                <constraints nullable="false" unique="true" uniqueConstraintName="file_upload_guid_uk"/>
+            </column>
+            <column name="bucket_file_url" type="varchar(1000)">
+                <constraints nullable="false" unique="true" uniqueConstraintName="bucket_file_url_uk"/>
+            </column>
+            <column name="url_creation_time" type="bigint">
+                <constraints nullable="false"/>
+            </column>
```

Comment:
We've been more recently using datetime(3) for dates with times. Check `InvitationSql` and `db-changes/schema/DDP-4266-invitation.xml`

Reply:
`InvitationSql` has actually been moved to `datetime(6)`. Let's use `datetime(6)` here.

Reply:
Done